dungeon-crawler/android_companion/src/main/java/com/novoda/dungeoncrawlercompanion/GameFragment.kt

```diff
@@ -0,0 +1,96 @@
+package com.novoda.dungeoncrawlercompanion
```

Comment:
we probably should have left the same java package name to make moving classes easier / less diff.  But not bothered now tbh sheep it :)

Reply:
need a separate package for the app itself tho, but yeah we can move some classes to the same package name as the other modules